### PR TITLE
Add numbers builtins

### PR DESCRIPTION
**Pull request by anderseknert**
Signed-off-by: Anders Eknert <anders@eknert.com>

### CODE REVIEW

File: src/main/clojure/jarl/builtins/numbers.clj [hunk 1]
```diff
@@ -0,0 +1,98 @@
+(ns jarl.builtins.numbers
+  (:require [jarl.builtins.utils :refer [check-args]])
+  (:import (se.fylling.jarl UndefinedException)))
+
+(defn- possibly-int
```
Comment (johanfylling) [inline src/main/clojure/jarl/builtins/numbers.clj:5]: Nice.

**Review by johanfylling — APPROVED** [review]

**Comment by johanfylling** [comment]
Huh, GH got a bit messed up. Or am I the only one seeing the double-postings?

**Comment by anderseknert** [comment]
> Or am I the only one seeing the double-postings?

Time to stop drinking, buddy 😄 No, I see it as well.

File: src/main/clojure/jarl/builtins/numbers.clj [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  [a b]
+  (check-args (meta #'builtin-mul) a b)
+  (possibly-int (* a b)))
+
+(defn builtin-div
+  "Implementation of div built-in"
+  {:builtin "div" :args-types ["number" "number"]}
+  [a b]
+  (check-args (meta #'builtin-div) a b)
+  (possibly-int (double (/ a b))))
```
[the remark this answers is not in the document]
Comment (anderseknert) [reply src/main/clojure/jarl/builtins/numbers.clj:38]: Good catch! I'll add handling of that.

[the remark this answers is not in the document]
Comment (anderseknert) [reply src/main/clojure/jarl/builtins/numbers.clj]: @johanfylling done

**Pull request merged**